README.md

```diff
@@ -33,22 +33,40 @@ LookerEmbedSDK.createDashboardWithId(11)
 
 A more complete example can be found [here](demo/demo.ts). Detailed instructions on how to use it are [here](#demo).
 
-## Details
+## Using the Looker Embed SDK  
 
-The Looker Embed SDK uses a fluent interface pattern. The construction of the embedded content is broken into two phases, building and connecting.
+### Web App Setup
+The Looker Embed SDK is a JavaScript module that lives and runs in the browser with your web app. 
+
+The demo also includes a reference implementation of the Looker SSO embed URL signature algorithm as a NodeJS web server module. The URL signature is used to secure and authenticate the embed URL request on the Looker server.
+ 
+The instructions and demo web app shown below assume that the Looker Embed SDK is built into your web app using the included webpack build configuration. If your web app doesn't use the webpack stack, you can build the Embed SDK into a javascript module and then reference that module from your web app HTML. 
```

Comment:
We don't assume Webpack. It's a UMD format module that should work with most Javascript module toolchains.